packages/p2p/package.json

```diff
@@ -31,11 +31,16 @@
     "license": "ISC",
```

Comment:
I guess you need to regenerate `package-lock.json` by running `npm run bootstrap:dev` in the root